src/api/controllers/verifyEmail.js

```diff
@@ -8,20 +8,22 @@ import errorMessages from '../../helpers/constants/error.messages';
  */
 export default class {
   /**
-     * @description verification link controller
-     * @param {*} req
-     * @param {*} res
-     * @returns {*} void
-     */
+   * @description verification link controller
+   * @param {*} req
+   * @param {*} res
+   * @returns {*} void
+   */
   static async verifyEmail(req, res) {
     const { token } = req.params;
-    const result = decodejwt(token);
-    if (result !== undefined) {
-      const action = await models.User.update({ verified: true }, {
-        where: {
-          email: result.user.email,
-        }
-      });
+    let result;
+    try {
+      result = decodejwt(token);
```

Comment:
@nkpremices hmm, these changes aren't related to your PR

Reply:
Actually after rebasing, one test of Daniel was not passing. 
I had to add a try catch here so that it could pass. 
I communicated with daniel @MemunaHaruna .